compiler/passes/scopeResolve.cpp

```diff
@@ -1702,7 +1702,19 @@ static void resolveEnumeratedTypes() {
       SET_LINENO(call);
 
       if (SymExpr* first = toSymExpr(call->get(1))) {
-        if (EnumType* type = toEnumType(first->symbol()->type)) {
+        // Go through chains like:
+        // enum color = { ... }
+        // type col = color;
+        // var a = col.red;
```

Comment:
I suggest changing the comment `var a = col.red;` to `which is needed to support e.g. 'col.red'`